interface/orders/orders_results.php

```diff
@@ -699,30 +699,30 @@ function validate(f) {
         "z-index:1000;left:-1000px;top:0px;font-size:9pt;'>\n" .
         "<table width='100%'>\n" .
         "<tr><td class='bold' align='center' colspan='2' style='padding:4pt 0 4pt 0'>" .
-        htmlspecialchars($result_text) .
+        text($result_text) .
         "</td></tr>\n" .
         "<tr><td class='bold' width='1%' nowrap>" . xlt('Status') . ": </td>" .
         "<td>" . generate_select_list(
             "form_result_status[$lino]",
             'proc_res_status',
             $result_status,
-            xl('Result Status'),
+            xlt('Result Status'),
```

Comment:
Note that should only by htmlescaping stuff right before it is sent to html output. Should not escape above since this is a function call (which will do it's own html escaping on the html output). So should revert above line.